config/initializers/devise.rb

```diff
@@ -248,7 +249,11 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-
+  ## OAuth integration Sep 4,2016 -- Shreyams
+  config.omniauth :facebook, "KEY", "SECRET"
+  config.omniauth :twitter, "KEY", "SECRET"
+  config.omniauth :linked_in, "KEY", "SECRET"
```

Comment:
This will not work in production. Look into methods of safe storage/deployment of the keys/secrets, such as environment variables.